dev/clint/src/clint/linter.py

```diff
@@ -455,6 +456,41 @@ def _check_forbidden_set_active_model_usage(
             rules.ForbiddenSetActiveModelUsage(),
         )
 
+    def _resolve_call(self, node: ast.AST) -> list[str]:
+        if isinstance(node, ast.Call):
+            return self._resolve_call(node.func)
+        elif isinstance(node, ast.Attribute):
+            return self._resolve_call(node.value) + [node.attr]
+        elif isinstance(node, ast.Name):
+            return [node.id]
+        return []
+
+    def _log_model_called_with_artifact_path(self, node: ast.Call) -> bool:
```

Comment:
Wondering how efficient this check is, is it slow?

Reply:
Good question. It's slow ∶)